public/locales/en-US/activate.ftl

```diff
@@ -447,12 +447,13 @@ webext-activity-resources-5 = <rLink>Extension Examples and Walkthroughs by Visw
 webext-activity-resources-6 = <rLink>Let’s Write a Web Extension by Blake Winton</rLink>
 webext-activity-resources-7 = <rLink>Create a Simple WebExtension by David Wals</rLink>
 webext-activity-resources-8 = <rLink>Browser Compatibility Table for APIs</rLink>
+webext-activity-resources-9 = <rLink>Webextensions.tech- a scaffolding tool for browser extensions</rLink> by Shivam Singhal, Tushar Arora &amp Trishul
```

Comment:
I think it would make sense to have a space before the - as well. What do you think? Also, can you use & instead of &amp; here?

Reply:
Done :+1: